src/styles/indentation.scss

```diff
@@ -0,0 +1,8 @@
+ol.os-raise-noindent {
```

Comment:
In terms of organizing these, what do you and @P-Gill97 think about the following options:

1) Creating a `content.scss` with various content style classes
2) Creating sort of a use case specific stylesheet 

The current PRs seem to follow the second approach, but I'm wondering if that's going to result in a lot of small style files which might get a bit confusing after a while :man_shrugging: . If we go with the former we could refactor if / when a single file becomes unwieldy?

Reply:
I see what you mean, I like the approach of having them together is an encompassing 'style' file and then breaking it apart later as need be.